test/integration/completed_transaction_test.rb

```diff
@@ -2,11 +2,32 @@
 require 'integration_test_helper'
 
 class CompletedTransactionTest < ActionDispatch::IntegrationTest
+  setup do
+    @payload = {
```

Comment:
We could remove some of the details here to make it easier to read, as I'm guessing we don't test for each one (eg analytics_identifier). Not sure what we've done previously for the other formats